hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreCompactor.java

```diff
@@ -100,15 +101,15 @@ public MemStoreCompactor(CompactingMemStore compactingMemStore) {
   public boolean start() throws IOException {
     if (!compactingMemStore.hasImmutableSegments()) return false;  // no compaction on empty
 
-    int t = compactingMemStore.getConfiguration().getInt(COMPACTING_MEMSTORE_TYPE_KEY,
+    int memStoreType = compactingMemStore.getConfiguration().getInt(COMPACTING_MEMSTORE_TYPE_KEY,
         COMPACTING_MEMSTORE_TYPE_DEFAULT);
 
-    switch (t) {
-      case 1: type = Type.COMPACT_TO_SKIPLIST_MAP;
+    switch (memStoreType) {
```

Comment:
so if we remove the compact to skip list option, what is left to do here is to set the action to the "default" behavior, say merging.
And this can be changed later by the policy.